numpy/core/_add_newdocs.py

```diff
@@ -1326,9 +1326,9 @@
 
     See Also
     --------
-    linspace : Evenly spaced numbers with careful handling of endpoints.
-    ogrid: Arrays of evenly spaced numbers in N-dimensions.
-    mgrid: Grid-shaped arrays of evenly spaced numbers in N-dimensions.
+    numpy.linspace : Evenly spaced numbers with careful handling of endpoints.
```

Comment:
This is weird. I think this fix, and the many like it, are indicating a bug crept into `numpydoc` or `add_newdoc`. This used to work, and while these changes fix warnings I'd prefer to solve the real issue instead.

Reply:
Perhaps it has to do with the `place` argument to `add_newdoc`. For example for this function, it uses `add_newdoc('numpy.core.multiarray', 'arange', ...)`. Would the issue be fixed if using `add_newdoc('numpy', 'arange', ...)` instead?

Reply:
I would call this a feature, not a bug. Explicit beats implicit, especially with documentation. For the on-line documentation the extra 6 letters don't matter, they are turned into a link. For the off-line documentation, a bare `linspace` can be confusing, especially when just getting started.

Reply:
This works in many places in the same file without the prefix. Example: https://numpy.org/devdocs/reference/generated/numpy.array.html#numpy.array

If there's a good reason it needs to be this way I have no problem with adding six characters, but I want to understand why something that used to work reliably (I think) now sometimes fails.

Reply:
Something is off. The [rendered documentation for this fix](https://numpy.org/devdocs/reference/generated/numpy.arange.html?highlight=arange#numpy.arange) is creating the links even without it. Maybe I overdid it with adding `numpy.` where not needed.

Reply:
I rebased without this commit, will look at it again later